### ISSUE TITLE
Custom PR header not working for monorepo w/o separate PRs.

### ISSUE THREAD

**Issue by sttobia** [google-github-actions/release-please-action#770]
### TL;DR

When having a monorepo with a `release-please-config.json`, the `pull-request-header` entry is only caught and propagated when the repo has `separate-pull-requests` set to `true`.

### Expected behavior

For my repos where I have the `release-please-config.json` that looks the following, everything works as expected and every new PR for the package gets opened with the custom header:
```
{
    "separate-pull-requests": true,
    "include-component-in-tag": true,
    "include-v-in-tag": true,
    "packages": {
        "some-package": {
            "pull-request-header": "Header: some custom header.",
            "package-name": "some-package-name",
            "release-type": "simple",
            "extra-files": [
                "Some/File.txt"
            ]
        }
    }
}
```

In another repo, where I use the following `release-please-config.json`, I would expect the same:

```
{
    "separate-pull-requests": false,
    "pull-request-header": "Header: some custom header.",
    "include-component-in-tag": true,
    "include-v-in-tag": true,
    "packages": {
         "some-package": {
            "package-name": "some-package-name",
            "release-type": "simple",
            "extra-files": [
                "Some/File.txt"
            ]
        }
    }
}
```

### Observed behavior

Instead, it still uses the default header being `:robot: I have created a release *beep* *boop*`.
It shows the same behavior, when I move the line into the `package` entry itself:

```
{
    "separate-pull-requests": false,
    "include-component-in-tag": true,
    "include-v-in-tag": true,
    "packages": {
         "some-package": {
            "pull-request-header": "Header: some custom header.",
            "package-name": "some-package-name",
            "release-type": "simple",
            "extra-files": [
                "Some/File.txt"
            ]
        }
    }
}
```

### Action YAML

```yaml
name: continuous delivery

on:
  push:
    branches:
      - main

jobs:
  release:
    runs-on: ubuntu-latest
    steps:
      - uses: google-github-actions/release-please-action@v3.7.8
        id: release
        with:
          command: manifest
          token: ${{secrets.GITHUB_TOKEN}}
          default-branch: main
```

### Log output

```text
Run google-github-actions/release-please-action@v3
❯ Fetching release-please-config.json from branch main
❯ Fetching .release-please-manifest.json from branch main
✔ Building releases
✔ Building strategies by path
❯ some-folder/Some.Library: simple
❯ some-folder/Some.Library.Foo: simple
❯ some-folder/Some.Library.Bar: simple
❯ Found pull request #47: 'chore: release main'
✔ Building release for path: some-folder/Some.Library
❯ type: simple
❯ targetBranch: main
⚠ pullRequestTitlePattern miss the part of '${scope}'
⚠ pullRequestTitlePattern miss the part of '${component}'
⚠ pullRequestTitlePattern miss the part of '${version}'
⚠ pullRequestTitlePattern miss the part of '${scope}'
⚠ pullRequestTitlePattern miss the part of '${component}'
⚠ pullRequestTitlePattern miss the part of '${version}'
✔ Building release for path: some-folder/Some.Library.Foo
❯ type: simple
❯ targetBranch: main
⚠ pullRequestTitlePattern miss the part of '${scope}'
⚠ pullRequestTitlePattern miss the part of '${component}'
⚠ pullRequestTitlePattern miss the part of '${version}'
⚠ pullRequestTitlePattern miss the part of '${scope}'
⚠ pullRequestTitlePattern miss the part of '${component}'
⚠ pullRequestTitlePattern miss the part of '${version}'
✔ Pull request contains releases, but not for component: some-library-foo
✔ Building release for path: some-folder/Some.Library.Bar
❯ type: simple
❯ targetBranch: main
⚠ pullRequestTitlePattern miss the part of '${scope}'
⚠ pullRequestTitlePattern miss the part of '${component}'
⚠ pullRequestTitlePattern miss the part of '${version}'
⚠ pullRequestTitlePattern miss the part of '${scope}'
⚠ pullRequestTitlePattern miss the part of '${component}'
⚠ pullRequestTitlePattern miss the part of '${version}'
✔ Pull request contains releases, but not for component: sime-library-bar
✔ Creating 1 releases for pull #47
❯ adding comment to https://github.com/mantis-tec/argos-backend-proxy/issues/47
❯ removing labels: autorelease: pending from issue/pull 47
❯ adding labels: autorelease: tagged from issue/pull 47
❯ Fetching release-please-config.json from branch main
❯ Fetching .release-please-manifest.json from branch main
✔ Building pull requests
✔ Building strategies by path
❯ some-folder/Some.Library: simple
❯ some-folder/Some.Library.Foo: simple
❯ some-folder/Some.Library.Bar: simple
✔ Collecting release commit SHAs
❯ release search depth: 400
❯ Fetching releases with cursor undefined
❯ Found release for path some-folder/Some.Library, some-library-v2.1.0
❯ Found release for path some-folder/Some.Library.Bar, sime-library-bar-v2.0.0
❯ Found release for path some-folder/Some.Library.Foo, some-library-foo-v1.0.1
❯ release for path: some-folder/Some.Library, version: 2.1.0, sha: 2f5cdbb64a7dde7363ab811f4e480ec2e458f07f
❯ release for path: some-folder/Some.Library.Bar, version: 2.0.0, sha: e6cb08bc180b10ec931b8b874025941ca344fece
❯ release for path: some-folder/Some.Library.Foo, version: 1.0.1, sha: 7d82da0b839f74b87ce0115c226a6a9d4ba9d71d
✔ Collecting commits since all latest releases
❯ commit search depth: 500
❯ Set(3) {
  '2f5cdbb64a7dde7363ab811f4e480ec2e458f07f',
  'e6cb08bc180b10ec931b8b874025941ca344fece',
  '7d82da0b839f74b87ce0115c226a6a9d4ba9d71d'
}
❯ Fetching merge commits on branch main with cursor: undefined
❯ Backfilling file list for commit: f8aa1ee77d0e2583c0c84b5eccb6075619a37eb9
❯ Found 1 files
❯ Backfilling file list for commit: 0436849799f1d91655c19336677d791502454555
❯ Found 1 files
❯ Backfilling file list for commit: db8f3055b639d0a7cc41b09c6cb968db1b8624fb
❯ Found 1 files
❯ Backfilling file list for commit: 31d05ea500dc452142a69e3c9b0f856facf02f79
❯ Found 1 files
❯ Backfilling file list for commit: 2002b9e4a67cf58c696a5b75df1e87cf775e7cf0
❯ Found 1 files
❯ Backfilling file list for commit: e233d9a5de8788ca6f04bb17c509a8f8b42ff07d
❯ Found 2 files
❯ Backfilling file list for commit: b02f652ceac4fa6b013035979277382232799655
❯ Found 1 files
❯ Backfilling file list for commit: a94cea506bb7b3b9ddb3bd6382fa5da5e3539c06
❯ Found 4 files
❯ Backfilling file list for commit: 08a1f47c24b236ffb566613052cf734d4c5bba68
❯ Found 10 files
❯ Backfilling file list for commit: cdcfa643f9c9b3d62697436a4676581a9f0ae216
❯ Found 14 files
❯ Backfilling file list for commit: 6edea5dad4b859a4de81792af98a024189b75902
❯ Found 2 files
❯ Backfilling file list for commit: 2e1fe3ee136ab8f2e4bd48a93c719f5a04090e41
❯ Found 3 files
✔ PR #34 has many files, backfilling
❯ Backfilling file list for commit: c24c5344b1afc9106c48f5b260aa04a4662abeda
❯ Found 124 files
❯ Backfilling file list for commit: 8eef5c78b9f96adfc1c568343bf5aad0ed647343
❯ Found 1 files
✔ Splitting 6 commits by path
✔ Building candidate release pull request for path: some-folder/Some.Library
❯ type: simple
❯ targetBranch: main
❯ commits: 0
✔ Considering: 0 commits
✔ No commits for path: some-folder/Some.Library, skipping
✔ Building candidate release pull request for path: some-folder/Some.Library.Foo
❯ type: simple
❯ targetBranch: main
❯ commits: 0
✔ Considering: 0 commits
✔ No commits for path: some-folder/Some.Library.Foo, skipping
✔ Building candidate release pull request for path: some-folder/Some.Library.Bar
❯ type: simple
❯ targetBranch: main
❯ commits: 0
✔ Considering: 0 commits
✔ No commits for path: some-folder/Some.Library.Bar, skipping
❯ running plugin: Merge
```

### Additional information

I am aware of [Issue-663](https://github.com/google-github-actions/release-please-action/issues/663), that's why I upgrade to the latest version `release-please-action@v3.7.8` which did not help.

**Comment by chingor13** [comment]
For advanced configuration options, please use a release-please-config.json (manifest config). https://github.com/googleapis/release-please/blob/main/docs/manifest-releaser.md
